bigquery/indexer.py

```diff
@@ -54,33 +49,25 @@ def parse_args():
     return parser.parse_args()
 
 
-def index_facet_field(es, index_name, primary_key, project_id, dataset_id,
-                      table_name, field_name, readable_field_name,
-                      billing_project_id):
+def index_table(es, index_name, primary_key, table_name, billing_project_id):
     """Indexes a facet field.
```

Comment:
`facet field` > `table`

Although I think it's pretty self-explanatory that the `index_table` function's purpose is to `Index a table`, so we can remove this line

Reply:
The part that's not obvious is table_name format (`<project id>.<dataset id>.<table name>`). That's why I have this pydoc.

Reply:
Ah, my reading of [Python style guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md) is that if you have pydoc for a method, it must have thedocstring part. Changed to table. (I had changed it before, but forgot to push.)